### PR TITLE
Lovepreet dev

### PR DESCRIPTION
**Pull request by Lovepreetcode4each**
1) css changes

### CODE REVIEW

File: app/Http/Controllers/UsersController.php [hunk 1]
```diff
@@ -70,80 +70,161 @@ public function register(UserRequest $request)
...
+    //             $data = $query->skip($start)->take($length)->get();
+    //             return response()->json([
+    //                 'data' => $data,
+    //                 'draw' => request()->input('draw', 1),
+    //                 'recordsTotal' => $totalRecords,
+    //                 'recordsFiltered' => $totalRecords,
+    //             ]);
+    //         } 
+    //         //--------------------------------------------------------------------------------------------
+    //         //  Show data  of user that is logged in
```
Comment (arsh91) [inline app/Http/Controllers/UsersController.php:124]: Please remove commented code.

File: resources/views/Basicgrade/allBasicGrade.blade.php [hunk 2]
```diff
@@ -161,6 +164,7 @@
     $(document).on("click", '#delete_btn', function(e) {
         e.preventDefault();
         var id = $(this).data('id');
```
Comment (arsh91) [inline resources/views/Basicgrade/allBasicGrade.blade.php:166]: Change the appropriate variable names.

File: app/Http/Controllers/UsersController.php (hunk 1 above)
Comment (Lovepreetcode4each) [reply app/Http/Controllers/UsersController.php]: done

Comment (Lovepreetcode4each) [reply app/Http/Controllers/UsersController.php]: done

**Review by arsh91 — APPROVED** [review]

**Pull request merged**